script/main.js

```diff
@@ -106,7 +106,7 @@ JFCustomWidget.subscribe("ready", function () {
         }
         document.getElementById('inp' + i).onchange = valueClick;
     }
-
+    JFCustomWidget.requestFrameResize({width: document.body.clientWidth, height: document.body.clientHeight})
```

Comment:
Following the last PR with the same property, this seems rather correct, doesn't it?
```suggestion
    JFCustomWidget.requestFrameResize({width: document.body.clientWidth, height: window.innerHeight})
```